cob_bringup/components/raw4_base.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+	<arg name="robot"/>
+	<arg name="can_device"/>
+
+	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+	<arg name="sim" default="false"/>
+
+	<arg name="driver_yaml" value="$(arg pkg_hardware_config)/robots/raw4/config/raw4_base_driver.yaml"/>
+	<arg name="controller_yaml" value="$(arg pkg_hardware_config)/robots/raw4/config/raw4_base_controller.yaml"/>
```

Comment:
are those files different for raw3 vs. raw4?
if not: stick with one single raw_base.launch